### DISCUSSION TITLE
Unable to install GRUB in /dev/nvme0n1

### DISCUSSION THREAD

**Discussion by ASIPDNJIOQWND** [ipxe/ipxe#1009] [Q&A]
Hi,

i have a problem after i set up the pxe server and install the ubunte i get this error 
(Unable to install GRUB in /dev/nvme0n1
executing 'grub-install /dev/nvme0n1' failed
This is a fatal error)
can someone help me

**Comment by NiKiZe** [comment]
iPXE is a network boot application.
It is not clear how this applies to your issue.
Nor is it clear what you have done, or where you are seeing this error.
